pyaedt/desktop.py

```diff
@@ -318,6 +301,9 @@ class Desktop:
 
     >>> desktop = Desktop("2021.1")
     >>> hfss = pyaedt.Hfss()
+    
+    The previous example initializes the `Desktop` module to the latest AEDT version 
+    installed on your machine in graphical mode and initializes HFSS.
```

Comment:
This section seems unnecessary.  Also, it's not correct as we're requesting the version of AEDT to be `"2021.1"`.  Let's remove this section.